CDCgov/prime-data-input-client
# Save test result to redux

# Issue
https://github.com/CDCgov/prime-central/issues/53

# About
- test results (positive/negative/inconclusive) and the corresponding device used are now saved to redux *after pressing submit*
- cleaned up / commented legacy code (i can't believe there is legacy code already...) from when we used to pull data from the data-input-api

# Not included
- success/failure alerts after submitting
- closing the test result after submitting

# How to test:
- play around with the app and submit things
- refresh to page to see that it persists only after you press submit

# Feedback Desired:
Scenario:
1. user adds patient to queue
2. user enters test result (ie: 'negative')
3. user clicks "submit"
4. patient disappears from queue automatically and success alert message appears

Q: what if, at this point, the user adds the patient back into the queue? Should the patient in the queue have the recently submitted data pre-populated, or should it be cleared and we can pretend its a new test? Is there some time period where we say, "this patient has already been given a test result in the past X hours" or something?
